operations/admin.go

```diff
@@ -103,7 +103,7 @@ func adminSetBanner() cli.Command {
 				return errors.Wrap(err, "loading configuration")
 			}
 
-			client, err := conf.setupRestCommunicator(ctx)
+			client, err := conf.setupRestCommunicator(ctx, false)
```

Comment:
Should this be false for the admin commands?

Reply:
Yeah I kind of feel like it's weird to print the banner when we're specifically updating it or getting settings or etc. Kind of arbitrary but since admin commands are mostly for us and banners are mostly for users it seems reasonable.